conductor-modules/arch-components-lifecycle/src/main/java/com/bluelinelabs/conductor/archlifecycle/ViewModelController.java

```diff
@@ -0,0 +1,18 @@
+package com.bluelinelabs.conductor.archlifecycle;
+
+import android.arch.lifecycle.ViewModelProvider;
+import android.arch.lifecycle.ViewModelProviders;
+import android.arch.lifecycle.ViewModelStore;
+
+public abstract class ViewModelController extends LifecycleController {
+
+    private ViewModelStore viewModelStore = new ViewModelStore();
+
+    protected ViewModelProvider viewModelProvider() {
```

Comment:
`new ViewModelProviders.DefaultFactory()` is deprecated. Can it be dropped here?

Reply:
Thanks! I did not realize 1.1.0 deprecated the method. It was replaced with https://developer.android.com/reference/android/arch/lifecycle/ViewModelProvider.AndroidViewModelFactory.html